marketplace.Makefile

```diff
@@ -5,6 +5,7 @@ __MARKETPLACE_MAKEFILE__ := included
 COMMIT ?= $(shell git rev-parse HEAD | fold -w 12 | head -n 1)
 
 makefile_dir := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+include $(makefile_dir)/gcloud.Makefile
```

Comment:
this was intended for applications only. Why did we need to add it here?

Reply:
We didn't - my mistake!